esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -1096,6 +933,7 @@ EsdlBindingImpl::EsdlBindingImpl()
 
 EsdlBindingImpl::EsdlBindingImpl(IPropertyTree* cfg, const char *binding,  const char *process) : CHttpSoapBinding(cfg, binding, process)
 {
+    m_pCentralStore.setown(createEsdlCentralStore());
```

Comment:
does createEsdlCentralStore return a singleton? if not, is there any reason why it can't be?

Reply:
Yes it's already a singleton.